Tests/TestCombine/CombineTests.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+class CombineTests: MockoloTestCase {
+
+    func testCombine() {
```

Comment:
Could you add a test with param `mockContent`? See `testSimpleInitParams` for an example.

Reply:
Test added. Please check to ensure it's testing what you were looking for.

Reply:
yup looks good